source/slang/slang.cpp

```diff
@@ -1032,9 +1032,17 @@ SLANG_NO_THROW slang::IModule* SLANG_MCALL Linkage::loadModuleFromSource(
         {
             return loadedModule;
         }
+        String pathStr = path;
+        auto pathInfo = PathInfo::makeFromString(pathStr);
+        if (File::exists(pathStr))
+        {
+            String cannonicalPath;
+            Path::getCanonical(pathStr, cannonicalPath);
```

Comment:
Path::getCanonical, can return an error. If that happens this could fall back to the old behavior?